surya/detection/affinity.py

```diff
@@ -74,7 +72,7 @@ def get_detected_lines(image, slope_tol_deg=2, vertical=False, horizontal=False)
                 bbox[1], bbox[3] = bbox[3], bbox[1]
             if bbox[2] < bbox[0]:
                 bbox[0], bbox[2] = bbox[2], bbox[0]
-            row = ColumnLine(bbox=bbox, vertical=vertical_line, horizontal=horizontal_line)
+            row = ColumnLine(polygon=bbox, vertical=vertical_line, horizontal=horizontal_line)
```

Comment:
same issue with polygonbox not taking in bboxes?

Reply:
Polygonbox can take either